test/jdk/java/lang/instrument/NativeMethodPrefixApp.java

```diff
@@ -96,6 +97,10 @@ private static void launchApp(final Path agentJar) throws Exception {
         final OutputAnalyzer oa = ProcessTools.executeTestJava(
                 "--enable-preview", // due to usage of ClassFile API PreviewFeature in the agent
                 "-javaagent:" + agentJar.toString(),
+                // we disable CheckIntrinsic because the NativeMethodPrefixAgent modifies
+                // the native method names, which then causes a failure in the VM check
+                // for the presence of an intrinsic on a @IntrinsicCandidate native method
+                "-XX:+UnlockDiagnosticVMOptions", "-XX:-CheckIntrinsics",
```

Comment:
Can you update both comments to begin with a capital letter and end with a period. Thanks.

Reply:
Done, I've updated the PR with this change.